git/merge-a-branch-as-a-single-commit.md

```diff
@@ -0,0 +1,7 @@
+# Merge all changes from another branch as a single commit:
+
+Let's say you want to merge `my-feature` (which contain 10 commits :D ) into `develop`:
+
+    git checkout develop
+    git merge --squash my-feature // Will stage new files for commit.
+    git commit -m "Merging my-feature branch"
```

Comment:
An anti-pattern? If you don't need to write a description, I think that is something perfectly valid.